src/utils.js

```diff
@@ -21,11 +21,11 @@ import log from './utils_log';
 import { version as apifyVersion } from '../package.json';
 
 // TYPE IMPORTS
-/* eslint-disable no-unused-vars,import/named,import/no-duplicates,import/order */
+/* eslint-disable no-unused-vars,import/named,import/no-duplicates,import/order,import/no-cycle */
```

Comment:
Hmm, yet another eslint statement that we didn't need before? You sure your ESLint config is correct?

Reply:
I don't mind the extra statement. It's just weird that we'd suddenly need it.

Reply:
yup, since I didn't reverted the previous commit, they stuck around, gonna remove them